tools/pip_install.sh

```diff
@@ -23,4 +23,6 @@ fi
 set -x
 
 # install the requested packages using the pinned requirements as constraints
-pip install -q --constraint <("$merge_reqs" "$dev_constraints" "$test_constraints") "$@"
+constraint_path="$(mktemp)"
+"$merge_reqs" "$dev_constraints" "$test_constraints" > "$constraint_path"
+pip install -q --constraint "$constraint_path" "$@"
```

Comment:
I guess that the tempfile should be deleted afterwards (like `[ -f "$constraint_path" ] && rm -f "$constraint_path"` or similar).